Tools/machines/frontier-olcf/submit.sh

```diff
@@ -45,5 +188,17 @@ export ROCFFT_RTC_CACHE_PATH=/dev/null
 export OMP_NUM_THREADS=1
 export WARPX_NMPI_PER_NODE=8
 export TOTAL_NMPI=$(( ${SLURM_JOB_NUM_NODES} * ${WARPX_NMPI_PER_NODE} ))
-srun -N${SLURM_JOB_NUM_NODES} -n${TOTAL_NMPI} --ntasks-per-node=${WARPX_NMPI_PER_NODE} \
-    ./warpx inputs > output.txt
+
+# _____________________________________________________________________________
+
+
+# ___ Run WarpX simulation ____________________________________________________
+
+if [ ! $is_pywarpx ]; then
```

Comment:
Invert the if-else so we do not need `!`?

```suggestion
if [ ! ${is_pywarpx} ]; then
```